src/content/time-tracking.mdx

```diff
@@ -0,0 +1,65 @@
+# Time Tracking Guide
+
+This tracking guide is not the same as general company’s rules. It relates only developer stuff and additional things which can be helpful for our work.
+For more projects/tasks details, please read our company tracking rules.
+
+## Where should I track my time?
+We are using a Harvest app to store tracked time. You can use a web / phone / desktop app or Chrome / Mozilla / Microsoft Edge extension [(look here)](https://www.getharvest.com/apps-and-integrations/directory#mobile-desktop). You can choose which one will be the best for you, to make it as easy as it’s possible & not disturb you while your daily stuff.
+
+
+## How should I track?
+
+### Project’s tasks
+It was intentionally described „Harvest app to store tracked time”, because it’s not always a place where you should add tasks.
+**To track any project’s task we are using a JIRA-Harvest connection**, which allows you to start and stop timer related to current task without your special involvement.
+The only thing you should do is to find a „Harvest Time Tracking by 42nd” in a task on JIRA and then connect your harvest to it. If you did it, you can just start timer directly from a task card and **it will be automatically visible in your harvest timesheet**, with proper project, task category and description with a link to the task added.
+The only thing you should do is to click big green button in JIRA :) What a time saving!
+
+Warning: please **don’t track project tasks without JIRA usage**. It’s really important to do it this way because of not just easy linking. JIRA should has updated timers in tasks to verify estimated time vs spend time. It’s possible only this way.
+
+### Project’s stuff
+Each project has a list of categories in Harvest, but only a development (or similar) is what you can track by JIRA connection. **All project things, which are not tasks you should track by Harvest, choosing the related category (meeting, code review, estimation etc.) and adding a description.** The last thing is really important to always be sure what meeting/estimation took how much time. Descripition of code review time is not obligatory.
+
+OMG, the expected project is not on the list in Harvest, what should I do?
+If you want to do a code review (or something else) for somebody from not-your project, please ask your PM to enable this project for you. It’s important to ask before starting doing this, because your PM should be informed about it and finally decide is it OK.
```

Comment:
```suggestion
If you want to do a code review (or something else) for somebody from not your project, please ask your PM to enable this project for you. It’s important to ask before starting doing this, because your PM should be informed about it and have a final say if it's ok.
```